src/platform/platform.base.js

```diff
@@ -21,7 +21,11 @@ export default class BasePlatform {
 	 * @returns {CanvasRenderingContext2D} context2d instance
 	 */
 	acquireContext(canvas, options) { // eslint-disable-line no-unused-vars
-		return undefined;
+		const context = canvas.getContext('2d');
```

Comment:
I'd change the return type (or remove it). Also would remove the whole body as function returns undefined by default.

Reply:
done